pkg/pyinst/bitmask.spec

```diff
@@ -33,6 +34,6 @@ if sys.platform.startswith("darwin"):
 		     name=os.path.join(
 		      'dist', 'Bitmask.app'),
                      appname='Bitmask',
-                     version='0.9.0rc2',
+                     version='0.9.2alpha1',
 		     icon='pkg/osx/bitmask.icns',
-		     bundle_identifier='bitmask-0.9.0rc2')
+		     bundle_identifier='bitmask-0.9.2alpha1')
```

Comment:
yes, I thought on this, but it needs a while to get it right.
the thing is that I have to fake it manually until the tag is done.